examples/step-5/step-5.cc

```diff
@@ -113,6 +113,14 @@ double coefficient (const Point<dim> &p)
   else
     return 1;
 }
+// @note It looks like we use a discontinuous coefficient, however,
+// the finite element method using quadrature really sees only a
+// polynomial approximation (defined by a
+// <code>quadrature_formula</code>) of that discontinuous coefficient
+// on every mesh. (Though that polynomial approximation of course
+// tends to a discontinuous function under mesh refinement.) Refer to
+// CellAccessor::set_material_id to set a discontinuous coefficient
```

Comment:
I think the text that's being proposed here is not quite accurate. It's not that an approximation of the coefficient is *formed*. What my answer on the mailing list wanted to indicate is the following: a quadrature formula only evaluates the integrand at a set of points; as a consequence, it cannot *distinguish* between a truly discontinuous function, and any other function that happens to have the same values at these points. That said, quadrature *approximates* the exact (but not computable) integral by the integral under a polynomial approximation that happens to coincide with the original integrand at the quadrature points.

So it isn't quite correct to say that "the coefficient as approximated on the cell is not strictly discontinuous" -- the coefficient is discontinuous alright, but the integrand is approximated by something that is a polynomial on each cell; of course, the approximating polynomial is discontinuous between cells.

All of this is complicated to explain. I'm not sure whether step-5 is the right place to try and explain this. But putting something into the results section of step-6 to explain why we care does make sense, and maybe something like the explanation above could be put there.